features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1157,6 +1157,7 @@
     -->
     <AdaptiveAuth>
    	    <ScriptEngine>{{AdaptiveAuth.ScriptEngine}}</ScriptEngine>
+   	    <UseThreadLocalScriptEngine>{{AdaptiveAuth.LimitScriptEngineCreation}}</UseThreadLocalScriptEngine>
```

Comment:
The config name is not updated

Reply:
Addressed with ad50ca4cbfef6afc884cfb89b4ad018ab4d3886c